packages/pipeline-editor/src/PipelineService.tsx

```diff
@@ -64,6 +64,15 @@ export class PipelineService {
     return metadataArr.find(r => r['name'] === name)['display_name'];
   }
 
+  /**
+   * The runtime name is currently based on the schema name (one schema per runtime)
+   * @param name
+   * @param metadataArr
+   */
+  static getRuntimeName(name: string, metadataArr: IDictionary<any>[]): string {
+    return metadataArr.find(r => r['name'] === name)['schema_name'];
+  }
```

Comment:
What is the schema for the local runtime?

Reply:
it will fallback to || local on the invocation of this method

Reply:
But doesn't the front-end need to the schema for certain operations?  What is the schema for the local runtime?

Reply:
there is no schema, and this "local runtime" is only visible for the run dialog...